### ISSUE TITLE
Set up netlify (PR preview is nice)

### ISSUE THREAD

**Issue by khxia** [uclaacm/hack-kahoot#22]

**Comment by TimothyGu** [comment]
This might be tricky if the frontend depends on a functional backend.

**Comment by mattxwang** [comment]
I heard [Heroku Review Apps](https://devcenter.heroku.com/articles/github-integration-review-apps) are pretty solid? Not sure how great that is for y'alls use-case tho 👀

**Comment by khxia** [comment]
Does netlify not run the backend for us? 
Also, when testing the frontend, we were thinking of just directly navigating to the URL endpoints ourselves; the calls to the backend will also be placeholders for now.

**Comment by mattxwang** [comment]
> Does netlify not run the backend for us?

Nah, Netlify is mostly just static site deployment / FE stuff; they have some Lambda-like functionality (and drop-ins for auth, etc.) but you'd have to bake that into your app.
